projects/sport-data-valley/manual_seeds/daily_responses.rb

```diff
@@ -122,6 +122,7 @@ def create_daily_response(person, timestamp)
       protsub ||= ProtocolSubscription.new(person: person, protocol: protocol)
 
       protsub.state = ProtocolSubscription::COMPLETED_STATE
+      protsub.external_identifier = '23'
```

Comment:
This looks like something very specific to what you have to troubleshoot. Remove?

Reply:
Sure. I'll comment it out.

Reply:
Fixed.